### PR TITLE
Improve the doc and add quickstart link

### PR DESCRIPTION
**Pull request by loicmathieu**
Related to the quickstart PR https://github.com/quarkusio/quarkus-quickstarts/pull/639

### CODE REVIEW

File: docs/src/main/asciidoc/funqy-gcp-functions.adoc [hunk 1]
```diff
@@ -40,6 +40,34 @@ At the time of this writing, Cloud Functions are still in beta so make sure to i
...
+    -Dextensions="funqy-google-cloud-functions"
+----
+
+Now, let's remove what's not needed inside the generated application:
+
+- Remove the dependency `io.quarkus:quarkus-reasteasy` from your `pom.xml` file.
+- Remove the generated `org.acme.quickstart.GreetingResource` class.
+- Remove the `index.html` from `resources/META-INF/resources` or it will be picked up instead of your Function.
+- Remove the existing tests.
+
```
Comment (machi1990) [inline docs/src/main/asciidoc/funqy-gcp-functions.adoc:70]: Looks like codestarts will help out here, right? 

@ia3andy @patriot1burke

Comment (loicmathieu) [reply docs/src/main/asciidoc/funqy-gcp-functions.adoc:70]: Yes, some day ...

Comment (ia3andy) [reply docs/src/main/asciidoc/funqy-gcp-functions.adoc:70]: @machi1990  Yes, you will just need to add `skipExample`.



> Yes, some day ...

@loicmathieu it shouldn't be that long before we make it default, probably 1.8.x @maxandersen?

Comment (machi1990) [reply docs/src/main/asciidoc/funqy-gcp-functions.adoc:70]: Thanks for the response. Can't wait till this becomes the default, as this paragraph in doc just shows the need for it.

Comment (loicmathieu) [reply docs/src/main/asciidoc/funqy-gcp-functions.adoc:70]: @ia3andy I means some day I will take a look at codestart and implement them for my extensions. 
I just didn't have that time in the near future

**Review by gsmet — APPROVED** [review]

**Pull request merged**